config/cucumber.yml

```diff
@@ -5,6 +5,6 @@ rerun_opts = rerun.to_s.strip.empty? ? "--format #{ENV['CUCUMBER_FORMAT'] || 'pr
 std_opts = "#{pretty} --tags 'not (@devready or @wip)'"
 %>
 default: <%= std_opts %> features
-wip: --tags @wip:3 --wip features
+wip: --tags @wip:5 --wip features
```

Comment:
WIP limit of 5 for 2 people? hmmm.....

Reply:
Running the outline scenarios count as one for each username

Reply:
We can change it back now that they aren’t in wip

Reply:
Ah. Just messing with you (mostly).